data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/reconciler/impl/ResourcesReconcilerMessageHandler.java

```diff
@@ -43,11 +43,15 @@ public ResourcesReconcilerMessageHandler(
   public void handle(Message<Object> event) {
     DataPlaneContract.Contract contract = (DataPlaneContract.Contract) event.body();
     resourcesReconciler.reconcile(contract.getResourcesList())
-      .onSuccess(
-        v -> logger.info("reconciled contract generation {}", keyValue("contractGeneration", contract.getGeneration())))
-      .onFailure(cause -> logger
-        .error("failed to reconcile contract generation {}", keyValue("contractGeneration", contract.getGeneration()),
-          cause));
+      .onSuccess(v -> logger.info(
+        "reconciled contract generation {}",
+        keyValue("contractGeneration", contract.getGeneration())
+      ))
+      .onFailure(cause -> logger.error(
+        "failed to reconcile contract generation {}",
+        keyValue("contractGeneration", contract.getGeneration()),
+        cause
+      ));
```

Comment:
This is just a style change, I'll revert it, but we should probably re-introduce checkstyle since editor config is not really keeping our style consistent.